collections-arrays/HowToEfficientlyRandomizeAnArray/HowToEfficientlyRandomizeAnArray/Program.cs

```diff
@@ -0,0 +1,12 @@
+﻿using BenchmarkDotNet.Running;
```

Comment:
Since this is targetting .NET 7.0 we can drastically simplify this by making use of top level statements.

Reply:
Moved to top-level statements.